x-pack/metricbeat/module/mssql/db/_meta/docs.asciidoc

```diff
@@ -0,0 +1 @@
+This is the db metricset of the module mssql.
```

Comment:
Should we add some more details here on what metrics are fetched?

Reply:
I can but it's going to be redundant with the in `fields.yml`, right? I want to be sure that you mean that information

Reply:
Yes. I should rephrase this to not necessarly about the metrics but that it describes it fetches metrics about DB and also potentially links to MSSQL docs where these metrics are. We can do this in a follow up.